src/main/scala/workflow/Node.scala

```diff
@@ -0,0 +1,24 @@
+package workflow
+
+import org.apache.spark.rdd.RDD
+
+sealed trait Node  {
+  def label: String = {
```

Comment:
If this is meant to be used for DAG drawing - we have to be careful with parameterization. For example, an NGramsTransformer(2) is different from an NGramsTransformer(1) - and we might have multiple linear mappers in a pipeline, etc. (these are parameterized by their input). Not sure the right way to handle this, though..